__tests__/endpoints.test.js

```diff
@@ -4,19 +4,44 @@ const db = require("../db/connection");
 
 afterAll(() => db.end());
 
-describe("CORE: GET /api/topics", () => {
+describe("GET /api/topics", () => {
     test("Retrieve topics from test database, should respond with 200 and an array", () => {
         return request(app)
             .get("/api/topics")
             .expect(200)
             .then(response => {
                 expect(Array.isArray(response.body.topics)).toBe(true);
```

Comment:
We don't need to explicitly check if the body is an array here, as the fact we are able to use a forEach on the response body, indicates that it is an array.
We should (however) check the length of the array if asserting inside a forEach, to ensure we get all topics back.